color.html

```diff
@@ -3,14 +3,16 @@
 <custom-style>
   <style>
     html {
+      --material-base-color: #fff;
+      --dark-theme-base-color: #000;
       /* Text colors */
       --material-body-text-color: var(--light-theme-text-color, rgba(0, 0, 0, 0.87));
       --material-secondary-text-color: var(--light-theme-secondary-color, rgba(0, 0, 0, 0.54));
       --material-disabled-text-color: var(--light-theme-disabled-color, rgba(0, 0, 0, 0.38));
 
       /* Primary colors */
       --material-primary-color: var(--primary-color, #2962FF);
-      --material-primary-contrast-color: var(--dark-theme-base-color, #fff);
+      --material-primary-contrast-color: var(--dark-theme-base-color, #000);
```

Comment:
This change breaks the primary style buttons, for example. It should remain white.